Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -93,7 +93,7 @@ try {
             $contents += "`$VerbosePreference = 'continue'"
         }
 
-        $contents += ". $PSScriptRoot\UpdatePSModulePath.ps1 $UpdatePSModulePathArgument"
+        $contents += ". `"$PSScriptRoot\UpdatePSModulePath.ps1`" $UpdatePSModulePathArgument"
```

Comment:
can you check if you can use single quote ( ' ) instead of double ( " )

Reply:
@chshrikh As suggested changed the (") to (')